src/index.html

```diff
@@ -3,11 +3,183 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link rel="icon" href="./images/icons/logo.png">
+    <title>Nothing (R)</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;700&family=Space+Mono&display=swap"
+      rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <div class="container container__header">
+      <header class="header">
+        <nav class="navigation">
+          <a class="navigation__items" href="#">
+            <img class="navigation__logo" src="./images/logo.png"
+              alt="logo nothing(r)"></img>
+          </a>
+          <div class="navigation__icons">
+            <a class="phone" href="tel:+0">
```

Comment:
Add the same phone number as in the span so it's consistent ;)

Reply:
yeah, I did it on purpose (like with address before) - the phone number is fake, so what is a difference?